src/transforms/babel.js

```diff
@@ -301,3 +307,20 @@ async function getJSXConfig(asset, isSourceModule) {
     };
   }
 }
+
+/**
+ * Generates a babel config for stripping away Flow types.
+ */
+function getFlowConfig(asset, isSourceModule) {
+  if (
+    !isSourceModule &&
```

Comment:
Should this be the inverse? Wouldn't we want to enable flow on app code and locally linked modules? I'm not sure enabling it on published node_modules is a good idea. WDYT?

Reply:
The issue is published code, as they shouldn't ship with flow types, local code should follow babelrc.

Shipped code should probably use flow comments or seperate .flow files

Reply:
I think it would be a nice feature to enable flow automatically on source code just like we do for JSX.

Shipped code should generally be pre-transpiled to remove the types I thought... Not sure, I guess it doesn't hurt to automatically do it for everything.

Reply:
It should, but apparently not every module does it as you can see in the linked issue